tools/run_tests/artifacts/build_artifact_python.sh

```diff
@@ -57,6 +58,7 @@ ${SETARCH_CMD} "${PYTHON}" setup.py sdist
 
 # Wheel has a bug where directories don't get excluded.
 # https://bitbucket.org/pypa/wheel/issues/99/cannot-exclude-directory
+# shellcheck disable=SC2086
 ${SETARCH_CMD} "${PYTHON}" setup.py bdist_wheel $WHEEL_PLAT_NAME_FLAG
```

Comment:
Double quote ([SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086)) seems applicable to this shell command. Quoting variables should fix the shellcheck complain.

Reply:
That wouldn't work. WHEEL_PLAT_NAME_FLAG can be empty and in that case `"$WHEEL_PLAT_NAME_FLAG"` becomes `""` and an empty arg will be forcibly passed to the command (which does break it).